components/Ink/src/private/MDCInkLayer.m

```diff
@@ -379,16 +432,40 @@ - (void)exit:(BOOL)animated {
   [self addAnimation:_backgroundOpacityAnim forKey:kInkLayerBackgroundOpacityAnim];
 }
 
+- (void)clearAnimations {
```

Comment:
@ianegordon suggested that we stick with the removeAllAnimations nomenclature and call super which I think is what we should do.